### ISSUE TITLE
use_bias can be set to False when dense or conv followed by batchnorm

### ISSUE THREAD

**Issue by chinakook** [yangyanli/PointCNN#21]
For example,
https://github.com/yangyanli/PointCNN/blob/f194cabd5c32fb454204b1cd507cee0af7c65ca6/pointfly.py#L283
Tensorflow's API use 'use_bias=True' as default.

**Comment by yangyanli** [comment]
This can be done by setting 'use_bias=not with_bn', right?

**Comment by chinakook** [comment]
Yes. This chang can speed up calculations without accuracy loss.

**Comment by yangyanli** [comment]
Thanks!

Fixed in c517171.
